guillotina/commands/__init__.py

```diff
@@ -16,13 +16,6 @@
 import yaml
 
 
-try:
-    import uvloop  # type: ignore
-
-    uvloop.install()
-except ImportError:
-    pass
-
```

Comment:
@bloodbare  this was causing some tests fails because it was changing the current loop.

I was about to move this to Command.run_command() but I found this: https://github.com/plone/guillotina/pull/788/files#diff-fdbd14b1d4ea412cb601947ddd0b74afL142

I'm not sure what to do

Reply:
Yes, we shouldn't install this automatically I think.  It should be in the bootstrap of the loop.

I didn't think this was even necessary if you set the loop policy setting.